src/sph-partition.cc

```diff
@@ -59,35 +61,62 @@ class Worker : public ChunkReducer {
 private:
     csv::Editor _editor;
     std::pair<int,int> _pos;
+    int _idField;
     int _chunkIdField;
     int _subChunkIdField;
+    std::string _idFieldName;
+    std::string _chunkIdFieldName;
+    std::string _subChunkIdFieldName;
     Chunker _chunker;
     std::vector<ChunkLocation> _locations;
+    bool _disableChunks;
 };
 
 Worker::Worker(po::variables_map const & vm) :
     ChunkReducer(vm),
     _editor(vm),
     _pos(-1, -1),
+    _idField(-1),
     _chunkIdField(-1),
     _subChunkIdField(-1),
-    _chunker(vm)
+     // disable overlap if not in the RA/DEC partitioning mode
+    _chunker(vm, vm.count("part.pos") == 0),
+    _disableChunks((vm.count("part.disable-chunks") != 0) && (vm["part.disable-chunks"].as<int>() != 0))
```

Comment:
I don't think you need `vm.count` test before getting the value, as it has a default value. And I think this option should a be a boolean flag and not integer value.

Reply:
Fixed